src/cli/query.py

```diff
@@ -1,5 +1,5 @@
 import os
-from dataregistry.db_basic import SCHEMA_VERSION, create_db_engine
```

Comment:
Dont need to import SCHEMA_VERSION anymore?

Reply:
No, I guess not.